web/libs/storybook/select/select.stories.tsx

```diff
@@ -143,6 +143,34 @@ const meta: Meta<typeof Select> = {
             {...args}
           />
         </div>
+        <div>
+          <Select
+            options={[
+              { label: "Yes", value: true },
+              { label: "No", value: false },
+            ]}
+            label="Boolean values with displayValueOverride"
+            displayValueOverride={(selectedOptions, placeholder) => {
```

Comment:
I would drop the `Override` bit of the prop to make it cleaner and less redundant

Reply:
the overRide is to indicate that its a function - instead of just a `displayValue` which indicate its a `string`, or some sort of html element

Reply:
could do displayValueHandler instead?

Reply:
`renderSelected` is great, thanks